docs/sdk/client-side-sdks/javascript/javascript-openfeature.md

```diff
@@ -23,15 +23,18 @@ DevCycle provides a Javascript implementation of the OpenFeature Web Provider in
 
 Install the OpenFeature Web SDK and DevCycle Web Provider:
 
-[//]: # (wizard-install-start)
+[//]: # 'wizard-install-start'
```

Comment:
hum... these were auto-formatted like this

Reply:
prettier has opinions. I don't question them

Reply:
yea I just wonder if it will work with the on-boarding stuff that uses this.

Reply:
onboarding uses the labs sdks - no the prod ones

Reply:
oh I see what you mean - one way to find out?

Reply:
looks like these need to be reverted: 
<img width="891" alt="image" src="https://github.com/user-attachments/assets/207f45d5-be1f-422b-9024-86d82a82f3f4">

Reply:
yup